website/blog/2022-05-26-aluminium.md

```diff
@@ -0,0 +1,246 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.6 - Aluminium
+authorURL: https://twitter.com/tanishiking25
+authorImageURL: https://github.com/tanishiking.png
+---
+
+We're happy to announce the release of Metals v0.11.6, which 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">119</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">46</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+## Reduce file watcher memory usage
+Previously, Metals uses `FileTreeRepository` of [swoval](https://github.com/swoval/swoval) for a file watcher, that maintains an in memory cache of some subset of the file system.
+Now, Metals uses `PathWatcher` of [swoval](https://github.com/swoval/swoval) instead. This implementation no longer caches the watched file trees, and Metals consumes smaller amount of memory.
+
+For more details, see the original PR: [\#3758](https://github.com/scalameta/metals/pull/3758)
+
+## (Scala3) Show scaladoc on hover for Scala 3 project
+Now, Metals shows the scaladoc on hover for Scala3 project. (Before this release, Metals unable to show the scaladoc for the symbols from third-party modules).
+
+![hover-scala3](https://i.imgur.com/Svzq5DD.png)
+
+### (Scala3) Support `completionItem/resolve`
+Now, in Scala 3 we get scaladoc and shows default values when moving accross completions. 
+
+![completion-item-resolve](https://i.imgur.com/Tz6AOsx.gif)
+
+## Show parent scaladoc if implementation is returning empty
+
+## Show scaladocs for signature help
+
+
+## Test explorer searches for test methods in parent classes
+
```

Comment:
@kpodsiad Could you have some words / screenshot for this feature?

Reply:
Sure ;)
```
## [MUnit] Test Explorer can find helper methods from parent classes

MUnit allows to use [helper functions](https://scalameta.org/munit/docs/tests.html#declare-tests-inside-a-helper-function) when declaring  tests. Very often those helper methods are extracted to some parent classes which are extended by many test suites.  
Now, Metals can find usages of those helper methods and display them in Test Explorer.  

This feature is available for Bloop and SBT 1.7.0-M2 or later.

![MUnit-helper-methods](https://imgur.com/GGRDpXA.gif)

## Support Cats Effect stacktraces in stacktrace analyzer

Cats Effect offers [asynchronous stack tracing](https://typelevel.org/cats-effect/docs/tracing#asynchronous-stack-tracing) which augments exceptions with additional information.  
Now, Stacktrace Analyzer is able to recognize CE's stacktraces and provide link to location in code. Say no to tedious debugging when you have only stacktrace from logs!

![cats-effect-stacktraces](https://imgur.com/5fMvcYd.gif)
```

Reply:
Added https://github.com/scalameta/metals/pull/3957/commits/719ca155165667a15a1dca92fb004eed5885e196 👍